sdk/tables/azure-data-tables/inc/azure/data/tables.hpp

```diff
@@ -13,7 +13,10 @@
 #include "azure/data/tables/credentials/named_key_credential.hpp"
 #include "azure/data/tables/dll_import_export.hpp"
 #include "azure/data/tables/enum_operators.hpp"
+#include "azure/data/tables/internal/constants.hpp"
 #include "azure/data/tables/models.hpp"
 #include "azure/data/tables/rtti.hpp"
-#include "azure/data/tables/tables_clients.hpp"
+#include "azure/data/tables/table_client.hpp"
 #include "azure/data/tables/tables_sas_builder.hpp"
+#include "azure/data/tables/table_service_client.hpp"
+#include "azure/data/tables/table_client_options.hpp"
```

Comment:
nit: add eof new line

```suggestion
#include "azure/data/tables/table_client_options.hpp"

```